neo/Consensus/ConsensusContext.cs

```diff
@@ -24,19 +25,20 @@ internal class ConsensusContext : IConsensusContext
         public ECPoint[] Validators { get; set; }
         public int MyIndex { get; set; }
         public uint PrimaryIndex { get; set; }
+        public ConsensusPayload[] ChangeViewPayloads { get; set; }
         public uint Timestamp { get; set; }
         public ulong Nonce { get; set; }
         public UInt160 NextConsensus { get; set; }
         public UInt256[] TransactionHashes { get; set; }
         public Dictionary<UInt256, Transaction> Transactions { get; set; }
```

Comment:
https://github.com/neo-project/neo/blob/519310a6fec6185d6408c0a27c166b05b1bc8fe8/neo/Consensus/ConsensusContext.cs#L28-L33

@erikzhang 
If we were to change `PrepareRequest` to be public we could include it here instead of having the 5 individual fields. However it makes the code a little longer throughout `ConsenusContext` having to include an extra `.prepareRequest` in a lot of places to access the fields; also perhaps it is nice to have it stay internal.

Reply:
```c#
public UInt160 NextConsensus => PreparationPayloads[MyIndex].GetDeserializedMessage<PrepareRequest>().NextConsensus;
```

Reply:
Nice idea.

Reply:
I think you meant `PrimaryIndex` instead of `MyIndex` though?

Reply:
A lot of orders of things would have to change in terms of order with Fill() and with parameters passed to make message types to be able to make the change to just have a getter and use the `PreparationPayloads[PrimaryIndex]`. Anyway. i'm going to resolve my own conversation here on this one.